### PR TITLE
Plat 1899 🙌 nav ft customer facing configs

### PR DESCRIPTION
**Pull request by nabinchha**
Jira: https://gretel.atlassian.net/jira/software/c/projects/PLAT/issues/PLAT-1899?filter=myopenissues&jql=project%20%3D%20%22PLAT%22%20AND%20assignee%20IN%20%28currentUser%28%29%29%20AND%20statusCategory%20IN%20%28%22To%20Do%22%2C%20%22In%20Progress%22%29%20ORDER%20BY%20created%20DESC

Opening it as a draft PR since it has dependencies on other PRs in monogretel to merge first. Hopfully CI checks are ignored.

### CODE REVIEW

File: config_templates/gretel/navigator_ft/navigator-ft-time-series.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+schema_version: "1.0"
+name: "navigator-ft-time-series" 
+models:
+  - navigator_ft:
+      data_source: __tmp__
+      group_training_examples_by: null # Set to column name of your choice
+      order_training_examples_by: null # Set to column name of your choice
```
Comment (nabinchha) [inline config_templates/gretel/navigator_ft/navigator-ft-time-series.yml:7]: Is this the right way of exposing this param that's dependent on the customer provided dataset?

File: config_templates/gretel/navigator_ft/navigator-ft-tabular.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+schema_version: "1.0"
```
Comment (nabinchha) [inline config_templates/gretel/navigator_ft/navigator-ft-tabular.yml:1]: should we rename this file to `default.yml`? I see this pattern but it doesn't seem to be consistent.

File: config_templates/gretel/navigator_ft/time-series.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+schema_version: "1.0"
+name: "time-series"
```
Comment (johnnygreco) [inline config_templates/gretel/navigator_ft/time-series.yml:2]: A couple thoughts:

- Maybe we should include `navigator-ft` in the name? This name will appear in the Console. Actually, I guess the full name would be `navigator_ft/tabular`? That works for the high-level SDK, but not sure about the Console. 

- I wonder if we should be more general than `time-series`. e.g., `sequential`? Ordering is also optional, but maybe we don't have to get into that.

**Review by johnnygreco — COMMENTED** [review]
Did we officially decide this should live in `gretel/navigator_ft` rather than `gretel/synthetics`?

File: config_templates/gretel/navigator_ft/navigator-ft-time-series.yml (hunk 1 above)
Comment (johnnygreco) [reply config_templates/gretel/navigator_ft/navigator-ft-time-series.yml]: Our other blueprint configs can be used directly without user input (this isn't completely true on all cases, though), so this new in the sense that they always have to provide input.

File: config_templates/gretel/navigator_ft/time-series.yml (hunk 3 above)
Comment (nabinchha) [reply config_templates/gretel/navigator_ft/time-series.yml:2]: - The pattern I see in other paths is to not include the top level name, so I think the full name will end up becoming `navigator_ft/tabular`
- time-series probably conveys the sequential nature? Also, isn't the goal of these blueprints to provide more specific examples?

**Comment by nabinchha** [comment]
> Did we officially decide this should live in `gretel/navigator_ft` rather than `gretel/synthetics`?

I don't think we ever talked about this. What do you think @sami-t?

File: config_templates/gretel/navigator_ft/time-series.yml (hunk 3 above)
Comment (johnnygreco) [reply config_templates/gretel/navigator_ft/time-series.yml:2]: The goal is to provide a simple default config that users can build upon. IMO "time series" is more specific than we want to be here. e.g., the ordered column doesn't necessarily have to be time (actually there doesn't even need to be an ordering at all, which is another consideration). I also think people put "time series"  and "event data" in separate bins in their mind, and "event data" is more aligned with what this model will be good at (at least initially). My thinking that "sequential" is general enough to encompass all ordered data.

**Comment by sami-t** [comment]
> > Did we officially decide this should live in `gretel/navigator_ft` rather than `gretel/synthetics`?
> 
> I don't think we ever talked about this. What do you think @sami-t?

@nabinchha Let's go with `gretel/synthetics/navigator-ft.yml` please. Thoughts on a separate time series blueprint vs. a single blueprint w/ comments showing how to change the config for time series? I'm leaning towards the latter, given time series will require config changes regardless (we can't currently provide a default time series blueprint that works out of the box).

**Comment by johnnygreco** [comment]
> > > Did we officially decide this should live in `gretel/navigator_ft` rather than `gretel/synthetics`?
> > 
> > 
> > I don't think we ever talked about this. What do you think @sami-t?
> 
> @nabinchha Let's go with `gretel/synthetics/navigator-ft.yml` please. Thoughts on a separate time series blueprint vs. a single blueprint w/ comments showing how to change the config for time series? I'm leaning towards the latter, given time series will require config changes regardless (we can't currently provide a default time series blueprint that works out of the box).

Good call on the single config @sami-t. And with the high-level SDK, it's super easy to add group / order by.

**Comment by nabinchha** [comment]
> > > Did we officially decide this should live in `gretel/navigator_ft` rather than `gretel/synthetics`?
> > 
> > 
> > I don't think we ever talked about this. What do you think @sami-t?
> 
> @nabinchha Let's go with `gretel/synthetics/navigator-ft.yml` please. Thoughts on a separate time series blueprint vs. a single blueprint w/ comments showing how to change the config for time series? I'm leaning towards the latter, given time series will require config changes regardless (we can't currently provide a default time series blueprint that works out of the box).

@sami-t this is done in 09ea9180e4be40a3a4248cb923fd7097a70876e5

**Review by sami-t — APPROVED** [review]

**Review by johnnygreco — APPROVED** [review]
🚀

**Pull request merged**